applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -359,6 +359,9 @@ def _GetInputType(self):
     def AddVariables(self):
         pass
 
+    def AddAdditionalHistoricalVariables(self,list_additional_variables):
```

Comment:
I would only add it if it is actually used

Reply:
yes... thing is that this is used as a solver in DEMApp but its not derived from python solver so I needed to add so it does no throw an error (same situation as AddVariable just above this)

Reply:
```suggestion
    def AddAdditionalHistoricalVariables(self, map_additional_variables):
```